### ISSUE TITLE
something should clarify

### ISSUE THREAD

**Issue by selous123** [SaoYan/IPMI2019-AttnMel#2]
##readme.md
In Section2(Data preparation)
[ISIC 2016]
File roots are different from what you actually used in your code

##in Paper
"For a fair comparison, we use the exact same training, validation, and test sets as were
provided in the challenge. "

I can only find train and test datasets in the challenge website [ISBI2016]
“https://challenge.kitware.com/#phase/56fc2763cad3a54f8bb80e51”
Can you provide any information about the validation set?

**Comment by SaoYan** [comment]
Hi,

Oops it seems we forget to clarify this in the paper.
ISIC2016 doesn't have the validation set, the model of the final epoch was taken for testing.
